pkg/services/store/entity/README.md

```diff
@@ -0,0 +1,140 @@
+# Unified Storage
+
+The unified storage projects aims to provide a simple and extensible backend to unify the way we store different objects within the Grafana app platform.
+
+It provides generic storage for k8s objects, and can store data either within dedicated tables in the main Grafana database, or in separate storage.
+
+By default it runs in-process within Grafana, but it can also be run as a standalone GRPC service (`storage-server`).
+
+## Storage Overview
+
+There are 2 main tables, the `entity` table stores a "current" view of the objects, and the `entity_history` table stores a record of each revision of a given object.
+
+## Running Unified Storage
+
+### Baseline configuration
+
+The minimum config settings required are:
+
+```ini
+; dev mode is required
+app_mode = development
+; need to specify target here for override to work later
+target = all
+
+[server]
+; https is required for kubectl
+protocol = https
+
+[feature_toggles]
+; enable unified storage
+unifiedStorage = true
+; enable k8s apiserver
+grafanaAPIServer = true
+; store playlists in k8s
+kubernetesPlaylists = true
+; store json id token in context
+idForwarding = true
```

Comment:
^^ is this required?

IIUC, it *is* required for the grpc flavor, but not required for in processs

Reply:
yes, technically it's only needed for grpc right now but it also doesn't hurt anything to have it enabled